docs/api/java/getting-started.md

```diff
@@ -80,23 +65,21 @@ import org.web3j.crypto.Credentials;
 
 public class Main {
     public static void main(String ...args) {
-        long chainId = 123L;// Chainid of the zksync network
+        long chainId = 123L;// Chainid of the ZkSync Era network
 
         Credentials credentials = Credentials.create("0x<private_key>");
 
         EthSigner signer = new PrivateKeyEthSigner(credentials, chainId);
     }
 }
-
 ```
 
-## Creating a wallet
-
-To control your account in zksync, use the `zksync.crypto.signer.EthSigner`. It can sign transactions with keys and send transactions to the zksync network.
+## ZkSync Era wallet
+You can connect the zkSync Era wallet for easier operations.
```

Comment:
This example is missing an import for `ZkSyncWallet`

Reply:
Thank you

Reply:
Hope you like this!!1